tools/c7n_tencentcloud/tests/conftest.py

```diff
@@ -0,0 +1,30 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+from c7n.config import Config
+from c7n.ctx import ExecutionContext
+from c7n_tencentcloud.client import Session
+
+
+@pytest.fixture
+def mock_env_aksk(monkeypatch):
+    monkeypatch.setenv("TENCENTCLOUD_SECRET_ID", "xxx")
```

Comment:
for the test environment variables on creds we generally use the test.env file in the root dir. that allows easier switch between actual and fake credentials when exercising tests.